.github/workflows/ci-workflow.yml

```diff
@@ -359,7 +359,6 @@ jobs:
         run: |
           ulimit -n 10240
           source /Users/runner/.cargo/env
-          cargo update -p cc --precise '1.0.83'
```

Comment:
I thought this was required? Could @benjaminwinger comment?

Reply:
Because of https://github.com/rust-lang/cc-rs/issues/900 being fixed, this is no longer required. A note on this is also in the commit message for logging.

Reply:
I see. Sorry I did not read it carefully.

Reply:
No worries :)